### DISCUSSION TITLE
Newbie who want to deploy its first app on cyber panel

### DISCUSSION THREAD

**Discussion by Danieljucke** [symfony/symfony#50067] [Q&A]
Hello I just started with symfony and now I'd like to deploy my app on cyber panel through hostinger 
I tried by myself and my root don't work same for my assets 
Can someone please help Me ? 
Thanks

**Comment by lyrixx** [comment]
Hello,

I suggest you to give more details. Like what you did, what did not work, what did you expect. Because ATM, it's hard to help you.

**Reply by Danieljucke** [reply]
Okay I compress the file then extract in my public_html folder 
I installed composer then made a composer update after that I installed the vendor and change into prod then I cleared the cache 
But the problem is my stylesheet and my images doesn't work and when I click the root doesn't work
